install.sh

```diff
@@ -2214,17 +1715,12 @@ ONEDB_PATCH_FILES="src/onedb/patches/4.14_monitoring.rb \
 
 ETC_FILES="share/etc/oned.conf \
            share/etc/defaultrc \
+           share/etc/guacd \
            src/tm_mad/tmrc \
-           src/scheduler/etc/sched.conf \
            src/monitor/etc/monitord.conf "
 
-EC2_ETC_FILES="src/vmm_mad/remotes/ec2/ec2_driver.conf \
-               src/vmm_mad/remotes/ec2/ec2_driver.default"
-
-AZ_ETC_FILES="src/vmm_mad/remotes/az/az_driver.conf \
-              src/vmm_mad/remotes/az/az_driver.default"
-
-VCENTER_ETC_FILES="src/vmm_mad/remotes/lib/vcenter_driver/vcenter_driver.default"
+SCHED_RANK_ETC_FILES="src/schedm_mad/remotes/rank/etc/rank.conf \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

SCHED_RANK_ETC_FILES appears unused. Verify use (or export if used externally).